### DISCUSSION TITLE
代码分支管理

### DISCUSSION THREAD

**Discussion by ghost** [Tencent/bk-ci#4238] [Ideas]
提个建议，每条流水线肯定要绑定一个代码库，那可以把选代码分支的步骤放在执行时设置参数的界面吗？而不是放在git插件里，只是建议！

**Comment by zanyzhao** [comment]
https://github.com/Tencent/bk-ci/issues/4240

可以关注下这个issue，会基于现有的流水线新增一类“必须绑定主代码库的CI流水线”
